model-optimizer/extensions/front/tf/CTCLossReplacement.py

```diff
@@ -55,17 +55,14 @@ def run_before(self):
     def pattern(self):
```

Comment:
please correct a comment for this transformation. It gets no longer valid due to seqlen format change.

Reply:
This is still unresolved. See right under class head.